arroyo/processing/processor.py

```diff
@@ -395,6 +400,10 @@ def _handle_invalid_message(self, exc: InvalidMessage) -> None:
     def _run_once(self) -> None:
         self.__metrics_buffer.incr_counter("arroyo.consumer.run.count", 1)
 
+        if time.time() - self.__last_run_log_ts >= LOGGING_FREQUENCY_SEC:
```

Comment:
please only calculate `time.time()` once, it's already very expensive to run it per `_run_once`

we also already have a counter metric for this, what is the purpose of the log?

Reply:
Per-consumer breakdown